lib/membership/membership-router.js

```diff
@@ -1,56 +1,75 @@
 'use strict';
 
 var express = require('express'),
-  config = require('./config-membership');
+  config = require('./config-membership'),
+  shortid = require('shortid');
 
-function initRouter(mediator) {
+var Topics = require('fh-wfm-mediator/lib/topics');
+
+function initRouter(topics) {
```

Comment:
`topics` is a bit generic. It should be more specific (e.g. `userDataCloudTopics`) or something like that.